index.html

```diff
@@ -51,48 +55,64 @@
                   JavaScript can be extended for a variety of purposes by
                   supplementing it with additional objects; for example:
                 </p>
+
                 <li>
+
                   - Client-side JavaScript extends the core language by
                   supplying objects to control a browser and its Document Object
                   Model (DOM). For example, client-side extensions allow an
                   application to place elements on an HTML form and respond to
                   user events such as mouse clicks, form input, and page
                   navigation.
+
                 </li>
+
                 <li>
+
                   - Server-side JavaScript extends the core language by
                   supplying objects relevant to running JavaScript on a server.
                   For example, server-side extensions allow an application to
                   communicate with a database, provide continuity of information
                   from one invocation to another of the application, or perform
                   file manipulations on a server.
+
                 </li>
+
+
               </article>
             </section>
             <br />
             <section class="main-section" id="What-you-should-already-know">
-              <header>What you should already know</header>
+              <header>
+                What you should already know
+              </header>
```

Comment:
- [ ] Kindly consider using a heading tag (h2, h3, ...) instead of the `header `tag for a better semantic. It would be great if you apply this advice to other <`header`> tags in your code 😉